app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt

```diff
@@ -585,16 +594,23 @@ object AchievementsModule {
             mapOf(
                 1 to links("openorienteeringmap")
             )
-        )
+        ),
+
+        Achievement(
+            QuestTypeAchievement.EMERGENCY.id,
+            R.drawable.ic_achievement_emergency,
+            R.string.achievement_emergency_title,
+            R.string.achievement_emergency_solved_X,
+            SolvedQuestsOfTypes,
+            // levels: 10, 30, 60, 100, 150, 210, 280, 360, 450, 550, 660, 780, 910, 1050, ...
+            { lvl -> (lvl + 1)*10 },
```

Comment:
Should it have an easier level ranking as they are perhaps rarer/more work/more valuable? e.g.
```suggestion
            // levels: 5, 10, 15, ...
            { lvl -> (lvl + 1)*5 },
```

Reply:
No, let's not start fine-tuning here, that's a can of worms. The only achievement that has a different progression is the "rare" achievement.